streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -61,18 +59,18 @@
 import org.apache.kafka.test.MockMetricsReporter;
 import org.apache.kafka.test.MockProcessorSupplier;
 import org.apache.kafka.test.TestUtils;
-import org.easymock.Capture;
-import org.easymock.EasyMock;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.After;
 import org.junit.rules.TestName;
-import org.junit.runner.RunWith;
-import org.powermock.api.easymock.PowerMock;
```

Comment:
Is `powermock` still used by streams module? If not, could you remove it from build.gradle?

Reply:
ditto